src/routes/home/components/CreateGameDialog.vue

```diff
@@ -18,31 +18,32 @@
       </v-btn>
     </template>
     <template #body>
-      <h4>
-        {{ t('home.playAiContent') }}
-        <a
-          class="text-cyan-lighten-2 text-decoration-none"
-          href="https://human-ai-interaction.github.io/cuttle-bot/"
-          target="_blank"
-        >
-          {{ t('home.playAiLink') }}
-        </a>
-        {{ t('home.playAiContent2') }}
-      </h4>
-      <form name="create_game_form" class="d-flex align-center">
-        <StatsScoringDialog
-          activator-color="surface-2"
-          :show-button-text="false"
-        />
-        <v-switch
-          v-model="isRanked"
-          class="d-flex align-center"
-          :label="isRanked ? t('global.ranked') : t('global.casual')"
-          data-cy="create-game-ranked-switch"
-          color="surface-2"
-        />
-      </form>
       <v-form @submit.prevent="submitNewGame">
+        <!-- V form now wraps the whole #body -->
```

Comment:
This comment is unnecessary

Reply:
I have removed this comment in latest commit.